lib/ansible/modules/uri.py

```diff
@@ -364,8 +364,13 @@
 # The return information includes all the HTTP headers in lower-case.
 content:
   description: The response body content.
-  returned: status not in status_code or return_content is true
+  returned: status not in status_code or return_content is true and reported Content-type is not "application/json"
```

Comment:
I'm not sure this addition is true, `content` should always be returned if `return_content: true`. Are you seeing different behaviour?

Reply:
Maybe I am wrong on this. That's just how I understood the description of the return_content parameter.

> Whether or not to return the body of the response as a "content" key in the dictionary result no matter it succeeded or failed.
> Independently of this option, if the reported Content-type is "application/json", then the JSON is always loaded into a key called json in the dictionary results.

Reply:
II just ran a quick test. You were right to mistrust this addition. I removed the addition to the description of the return property and instead added a little clarification to the description of the return_content parameter.